src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,24 @@
 package practice;
 
-public class CandidateValidator {
+import java.util.Arrays;
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
     //write your code here
+    private static final int MINIMUM_AGE = 35;
```

Comment:
```suggestion
    private static final int MIN_AGE = 35;
```
to be consistent with other constants

Reply:
done